integration-tests/tests/src/tests/alias.ts

```diff
@@ -51,9 +53,13 @@ function addTestObjects(realm) {
   realm.commitTransaction();
 }
 
-module.exports = {
-  testAliasInSchema() {
-    const Person = {
+describe("Realm Alias", () => {
```

Comment:
🔜 I think a more descriptive name might help newcomers understand the purpose of the tests

```suggestion
describe("Aliasing property names using mapTo", () => {
```